invoice.py

```diff
@@ -207,6 +213,21 @@ def payment_details(self):
                 if ml.account.type.receivable],
             key=attrgetter('maturity_date'))
 
+    @classmethod
+    def post(cls, invoices):
+        Config = Pool().get('account.configuration')
+
+        config = Config(1)
+        transaction = Transaction()
+        context = transaction.context
+
+        super(Invoice, cls).post(invoices)
+
+        with transaction.set_context(
+                queue_scheduled_at=config.invoice_facturae_after,
```

Comment:
AFAIU we don't want to delay the generation of the facturae document but the sending process.

Reply:
322fca81de602b0681b7d8b7312e27cd0e260ec0